src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/EntryPoint.kt

```diff
@@ -16,8 +20,10 @@ fun main(args: Array<String>) {
 }
 
 class Application {
+    private val defaultLogConfigFactory = LogConfigurationFactory(LogConfiguration())
+
     fun tryRunning(args: Array<String>) {
-        setConfigurationFactory(LogConfigurationFactory())
+        setConfigurationFactory(defaultLogConfigFactory)
```

Comment:
I was worried about the "chicken or the egg" problem, but using a default pre-parsing solves it nicely
Well done 🙂